05-expressions.tex

```diff
@@ -220,6 +220,12 @@ \section{for}
 
 Variable \expr{v} is then available within loop body \expr{e2} and holds the value of the individual elements of collection \expr{e1}.
 
+The closest Haxe for-loops come to C-style for-loops is probably using the range operator \expr{...}. It is a binary operator taking two \expr{Int} arguments: \expr{min...max} returns an \expr{IntIterator} instance that iterates from \expr{min} (inclusive) to \expr{max} (exclusive). Note that \expr{max} may not be smaller than \expr{min}.
```

Comment:
That first sentence is really out of place there. Integer iteration is only a special case of C-for-loops so I don't see why we should mention them at all (maybe they would make for a nice trivia?). Also "probably" isn't good in technical documentation.

Other than that:

* please use `\type{Int}`
* operators typically do not have arguments but operands
* expressions don't really "return" something. It is more correct to say that this syntax is typed to construct an instance of IntIterator or something.